### ISSUE TITLE
Add functionality to show a cropped preview of an image attached to a post

### ISSUE THREAD

**Issue by gone-still** [MarshalX/atproto#495]
When attaching an image to a post using the bluesky web client, there’s an option to crop the preview image that will appear in the post, showing a sub-region of the full image. Once the image is clicked, the full version is displayed. 

Can a similar “crop preview” functionality be added? Maybe by defining a bounding rectangle (sub matrix) of the original image?

**Comment by MarshalX** [comment]
I do not think that it would be part of the atproto SDK someday. Pillow lib will fit perfectly for your needs
